lms/djangoapps/student_account/views.py

```diff
@@ -294,3 +303,96 @@ def _external_auth_intercept(request, mode):
         return external_auth_login(request)
     elif mode == "register":
         return external_auth_register(request)
+
+
+@login_required
+@require_http_methods(['GET'])
+def account_settings(request):
+    """Render the current user's account settings page.
+
+    Args:
+        request (HttpRequest)
+
+    Returns:
+        HttpResponse: 200 if the page was sent successfully
+        HttpResponse: 302 if not logged in (redirect to login page)
+        HttpResponse: 405 if using an unsupported HTTP method
+
+    Example usage:
+
+        GET /account/settings
+
+    """
+    return render_to_response('student_account/account_settings.html', account_settings_context(request))
+
+
+def account_settings_context(request):
+    """ Context for the account settings page.
+
+    Args:
+        request: The request object.
+
+    Returns:
+        dict
+
+    """
+    user = request.user
+
+    country_options = [
+        (country_code, unicode(country_name))
+        for country_code, country_name in sorted(
+            countries.countries, key=lambda(__, name): unicode(name)
+        )
+    ]
+
+    year_of_birth_options = [(unicode(year), unicode(year)) for year in UserProfile.VALID_YEARS]
```

Comment:
@symbolist I'm having some issues with eventing caused by the fact that we change year_of_birth from an integer to a string. For example, after loading the page and changing the year_of_birth, the old_value that is evented is an integer (from the server), but the new_value is a string.

Would it be possible to create an IntegerDropdownFieldView so that we always maintain the year as an integer? The I'm doing the client-side eventing in saveAttributes, so I don't have knowledge of what the specific field is.